eclass/cvs.eclass

```diff
@@ -409,28 +376,29 @@ EOF
 				echo "${ECVS_SSH_HOST_KEY}" > "${known_hosts_file}" || die
 			fi
 
-			echo -n "newarglist.insert(1, '-oStrictHostKeyChecking=" \
-				>> "${CVS_RSH}" || die
-			echo "${strict_host_key_checking}')" \
-				>> "${CVS_RSH}" || die
-			echo "os.execv('${EPREFIX}/usr/bin/ssh', newarglist)" \
-				>> "${CVS_RSH}" || die
-
+			# Create a wrapper script to pass additional options to SSH
+			local -x CVS_RSH="${T}/cvs_sshwrapper"
+			cat > "${CVS_RSH}" <<-EOF || die
+				#!${BROOT}/bin/bash
+				exec "${BROOT}/usr/bin/ssh" \\
```

Comment:
This doesn't need an eprefixed bash, posix sh seems fine...

It does look like we dropped a couple of comments which in theory we could do using arrays, as those support comments, but the actual comments themselves look like the "add 1 and 1 together" type of comments so no actual point...

Reply:
I had an array first, but then decided that it's better readable without. Not much value in these comments either.